department-of-veterans-affairs/caseflow
# Hearing Prep | Show only current number of issues on Daily Docket

Resolves #4777

### Description
This PR shows only current number of issues on Daily Docket.

### Acceptance Criteria 
- [x] Show the count for current issues on appeal.
If a Judge has 2 current issues, and 7 prior issues, only show "2 issues" on Daily Docket.

### Testing Plan
1. Go to Daily Docket, check if only current issues are returned.
2. Go to Worksheet and compare current issues on daily docket and worksheet

<img width="1108" alt="screen shot 2018-04-11 at 1 06 24 pm" src="https://user-images.githubusercontent.com/23080951/38632005-6757339e-3d89-11e8-9902-59972cb5c836.png">


<img width="411" alt="screen shot 2018-04-11 at 11 50 54 am" src="https://user-images.githubusercontent.com/23080951/38631964-42a386b0-3d89-11e8-87d8-740567738361.png">
